src/gazebo_motor_model.cpp

```diff
@@ -248,5 +255,28 @@ void GazeboMotorModel::UpdateForcesAndMoments() {
 #endif /* if 0 */
 }
 
+/// \brief A function to check the motor_Failure_Number_ and stimulate motor fail
```

Comment:
the comment doc should be on the `gazebo_motor_model.h` (header file). In this case, above https://github.com/PX4/sitl_gazebo/pull/140/files#diff-8892233ba37c1996abc20f1a8ba5619cR152